src/test/rgw/rgw_multi/tests_es.py

```diff
@@ -77,7 +78,7 @@ def init_env(create_obj, num_keys = 5, buckets_per_zone = 1, bucket_init_cb = No
 
     # don't wait for meta sync just yet
     for zone, bucket in zone_bucket:
-        for count in xrange(0, num_keys):
+        for count in range(num_keys):
```

Comment:
default num_keys is 5, not sure it ever more then 1000 or so... no need to use xrange here.

Reply:
@kshtsk i don't want to take the risk. if we are not sure, why would we want to do this?

Reply:
please also see my reply at https://github.com/ceph/ceph/pull/34389#discussion_r403484551 . again, what i am aiming is the ultimate goal. personally, i don't really want take the risk or even the time to figure out the upper bound of `num_keys`.

Reply:
ok